workshops/arrow.qmd

```diff
@@ -28,11 +26,17 @@ This course is for you if you:
 
 -   want to learn how to engineer your tabular data storage for more performant access and analysis with Apache Arrow
 
+This course is probably not for you if:
```

Comment:
I feel like what the workshop is about is pretty well captured above. Once we start a list of what a course is not I think it opens a can of worms, and feels a little pff putting?

Reply:
Yeah, agree on offputting. Reflecting on the reasons I made the change, I think it can be achieved other ways.